Shared/Samples/Apply scene property expressions/ApplyScenePropertyExpressionsView.swift

```diff
@@ -0,0 +1,127 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyScenePropertyExpressionsView: View {
+    /// The scene that displays in the scene view.
+    @State private var scene: ArcGIS.Scene = {
+        let scene = ArcGIS.Scene(basemapStyle: .arcGISImageryStandard)
+        
+        // Give the scene an initial viewpoint.
+        let point = Point(x: 83.9, y: 28.4, z: 1000, spatialReference: .wgs84)
+        scene.initialViewpoint = Viewpoint(
+            latitude: point.y,
+            longitude: point.x,
+            scale: 0,
```

Comment:
```suggestion
            latitude: .nan,
            longitude: .nan,
            scale: .nan,
```
For scene view initial camera, in samples we typically use `nan` to convey that these values have no effect on the camera.